packages/theme/src/cli/utilities/theme-fs.test.ts

```diff
@@ -157,12 +170,17 @@ describe('theme-fs', () => {
       const root = 'src/cli/utilities/fixtures'
       const key = 'templates/404.json'
       const themeFileSystem = await mountThemeFileSystem(root)
-      expect(themeFileSystem.files.get(key)).toEqual({
+      await themeFileSystem.ready()
+      const file = themeFileSystem.files.get(key)
+      expect(file).toEqual({
         key: 'templates/404.json',
         checksum: 'f14a0bd594f4fee47b13fc09543098ff',
+        value: expect.any(String),
+        attachment: '',
       })
 
       // When
+      delete file?.value
       const content = await themeFileSystem.read(key)
```

Comment:
I had to update these tests because we now store `value` and `attachment` earlier (for free). I'm deleting the value to keep testing that `read` adds it back.